### ISSUE TITLE
sigsegv when trying to change security_group using _patch

### ISSUE THREAD

**Issue by 6illes** [scaleway/scaleway-cli#435]
tested server  _running_  and server _stopped_

```
> gilles@debian:~/scripts$ scw -D _patch xxxxxxxx-c58c-449f-9f73-dcbcfa72f51a security_group=xxxxxxxx-d39e-4d9b-ac66-244b398f3f0e
> DEBU[0000] curl -X HEAD -H "Content-Type: application/json" -H "User-Agent: scw/v1.11.1+dev" -H "X-Auth-Token: 00000000-0000-4000-8000-000000000000" 'https://cp-par1.s$
> DEBU[0000] [HEAD]: https://cp-par1.scaleway.com/servers/xxxxxxxx-c58c-449f-9f73-dcbcfa72f51a?
> DEBU[0000] curl -X GET -H "Content-Type: application/json" -H "User-Agent: scw/v1.11.1+dev" -H "X-Auth-Token: 00000000-0000-4000-8000-000000000000" 'https://cp-par1.sc$
> DEBU[0000] [GET]: https://cp-par1.scaleway.com/servers/xxxxxxxx-c58c-449f-9f73-dcbcfa72f51a
> DEBU[0000] [Response]: [200]
> {"server": {"state_detail": "booted", "image": {"default_bootscript": null, "creation_date": "2016-11-30T19:21:58.774950+00:00", "name": "JessieUTD", "modification_dat$
> DEBU[0000] security_group=yyyyyyyy-64f0-4b60-8981-3c0794c54739  =>  security_group=xxxxxxxx-d39e-4d9b-ac66-244b398f3f0e
> panic: runtime error: invalid memory address or nil pointer dereference
> [signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x47f34c]
> 
> goroutine 1 [running]:
> panic(0x7c2100, 0xc4200100e0)
>         /home/gilles/go/src/runtime/panic.go:500 +0x1a1
> github.com/scaleway/scaleway-cli/pkg/cli.runPatch(0xa02400, 0xc42000c1c0, 0x2, 0x2, 0x1, 0x0)
>         /home/gilles/gogo/src/github.com/scaleway/scaleway-cli/pkg/cli/x_patch.go:90 +0x10dc
> github.com/scaleway/scaleway-cli/pkg/cli.Start(0xc42000c1a0, 0x4, 0x4, 0xc420108480, 0x0, 0xc4200fec20, 0xc4200fec00)
>         /home/gilles/gogo/src/github.com/scaleway/scaleway-cli/pkg/cli/main.go:123 +0x7ee
> main.main()
>         /home/gilles/gogo/src/github.com/scaleway/scaleway-cli/cmd/scw/main.go:19 +0x83
> gilles@debian:~/scripts$
```

**Comment by QuentinPerez** [comment]
Thank you @6illes 😊

**Comment by 6illes** [comment]
I'm trying to fix it, will tell you

**Comment by QuentinPerez** [comment]
It's already fixed #436

**Comment by 6illes** [comment]
tested ok thanks
